src/main/java/budgetbuddy/commons/util/AppUtil.java

```diff
@@ -15,6 +17,16 @@ public static Image getImage(String imagePath) {
         return new Image(MainApp.class.getResourceAsStream(imagePath));
     }
 
+    /**
+     * Returns a {@code SimpleDateFormat} for use in displaying/parsing dates.
+     * The primary purpose of this method is to standardise date display across the app.
+     */
+    public static SimpleDateFormat getDateFormat() {
+        SimpleDateFormat format = new SimpleDateFormat("d/M/yy");
```

Comment:
Performance consideration: maybe share one `SimpleDateFormat`, instead of constructing a new one each time this is called?

[`ThreadLocal`](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/ThreadLocal.html) would be useful here (`SimpleDateFormat` is not thread-safe).